src/Request.php

```diff
@@ -1076,10 +1076,11 @@ private static function limitTelegramRequests($action, array $data = [])
 
                     $requests = DB::getTelegramRequestCount($chat_id, $inline_message_id);
 
-                    if ($requests['LIMIT_PER_SEC'] == 0     // No more than one message per second inside a particular chat
```

Comment:
@jacklul I'm looking at this again, as I'm setting up to release 0.45.0, and I really don't think that the `if` statement does the same thing as before.
Is this intentional to fix something?